packages/core/src/FileUploader/FileUploader.d.ts

```diff
@@ -18,6 +18,10 @@ export type HvFileUploaderClassKey = "root";
 
 export interface FileUploaderProps
   extends StandardProps<React.HTMLAttributes<HTMLDivElement>, HvFileUploaderClassKey> {
+  /**
+   * Id to be applied to the root node.
+   */
+  id: string;
```

Comment:
This won't lead to a warning, it will lead to a compilation error

Reply:
I opened an issue regarding the `id`s and `key`s: https://github.com/lumada-design/hv-uikit-react/issues/2873